modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/change/tracking/spi/reference/DDMFormInstanceTableReferenceDefinition.java

```diff
@@ -71,6 +72,12 @@ public void defineChildTableReferences(
 						DDMFormInstance.class.getName())
 				)
 			)
+		).singleColumnReference(
+			DDMFormInstanceTable.INSTANCE.structureId,
```

Comment:
A column should only map to one other column.  You are mapping structureId twice.  The one to DDMStructureVersion is not needed